src/test/counterComments.test.js

```diff
@@ -0,0 +1,25 @@
+const { counterComment } = require('./counterComments.js');
+
+const items = [{ comment: 'Hi', creation_date: '2023-05-24', username: 'Ali' },
```

Comment:
- [ ] For this mock, don't you think it would be better if the name variable is more representative for this test? For example: `sixItems`, instead of `items`.[OPTIONAL]